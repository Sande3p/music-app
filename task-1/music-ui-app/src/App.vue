<template>
  <div>
    <header class="main-header">
      <h3>Muisc Library</h3>
    </header>
    <main class="page">
      <Albums />
    </main>
  </div>
</template>

<script>
import Albums from "./components/Albums"

export default {
  name: "app",
  components: {
    Albums,
  },
};
</script>
