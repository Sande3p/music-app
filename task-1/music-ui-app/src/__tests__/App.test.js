import { shallowMount } from "@vue/test-utils";
import App from '../App';

describe('Albums.test.js', () => {
  let cmp;

  beforeEach(async () => {
    cmp = shallowMount(App)
  });

  it("has the expected html structure", () => {
    expect(cmp.element).toMatchSnapshot();
  });


});