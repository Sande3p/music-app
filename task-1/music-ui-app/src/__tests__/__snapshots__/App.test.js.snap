// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Albums.test.js has the expected html structure 1`] = `
<div>
  <header
    class="main-header"
  >
    <h3>
      Muisc Library
    </h3>
  </header>
   
  <main
    class="page"
  >
    <albums-stub />
  </main>
</div>
`;
